package.json

```
@@ -19,17 +20,21 @@
     "curves"
   ],
   "scripts": {
-    "rollup": "run-s rollup:docs rollup:dist",
-    "rollup:base": "rollup ./lib/bezier.js --format esm --file ",
-    "rollup:docs": "npm run rollup:base -- ./docs/js/bezier.js",
-    "rollup:dist": "npm run rollup:base -- ./dist/bezier.js",
+    "build": "run-s build:docs build:dist build:cjs",
+    "build:base": "rollup ./lib/bezier.js --format esm --file ",
```

this is just the rollup base command, calling it `rollup` makes the other build steps far easier to run.

```suggestion
    "rollup": "rollup ./lib/bezier.js --format esm --file ",
```